.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,20 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Issue with .NET runtime or core .NET libraries
+    url:  https://github.com/dotnet/runtime/issues/new/choose
+    about: Please open issues with the .NET runtime or core .NET libraries in their repo
+  - name: Issue with .NET Core SDK
+    url:  https://github.com/dotnet/sdk/issues/new/choose
+    about: Please open issues with the .NET Core SDK in their repo
+  - name: Issue with Entity Framework
+    url:  https://github.com/dotnet/efcore/issues/new/choose
+    about: Please open issues with Entity Framework in their repo
+  - name: Issue with Roslyn compiler
+    url:  https://github.com/dotnet/roslyn/issues/new/choose
+    about: Please open issues with the Roslyn compiler in their repo
+  - name: Issue with Windows Forms
```

Comment:
While I like the idea about adding links to EF and .NET Runtime here, I don't think having WinForms or WPF is reasonable.

We've never seen an issue related to these areas, so there is no data suggesting that this is helpful. So this may feel a bit more confusing, to have too many unrelated items in the list. Let's simply leave these out.

Reply:
Happy to